p2p/raintree/network.go

```diff
@@ -75,15 +79,11 @@ func (n *rainTreeNetwork) networkBroadcastAtLevel(data []byte, level uint32, non
 		return err
 	}
 
-	if addr1, ok := n.getFirstTargetAddr(level); ok {
-		if err = n.networkSendInternal(bz, addr1); err != nil {
-			log.Println("Error sending to peer during broadcast: ", err)
-		}
-	}
-
-	if addr2, ok := n.getSecondTargetAddr(level); ok {
-		if err = n.networkSendInternal(bz, addr2); err != nil {
-			log.Println("Error sending to peer during broadcast: ", err)
+	for _, target := range n.router.GetTargetsAtLevel(level) {
+		if target.ShouldSendInternal() {
```

Comment:
tl;dr Rename `ShouldSendInternal` to `isSelf`.

The goal of `networkSendInternal` was to be an internal helper, but the goal of the if should simply be: "Don't send a network message if I'm the target"